nntrainer/tensor/blas_interface.h

```diff
@@ -175,6 +175,14 @@ void ewva(const unsigned int N, const _FP16 *X, const _FP16 *Y, _FP16 *Z);
  * @param[in] X __fp16 * for Vector X
  */
 unsigned int isamax(const unsigned int N, const _FP16 *X, const int incX);
+
+/**
+ * @brief squared root transformation inplace : X = sqrt(X)
```

Comment:
I'm confused. is this func $X = \sqrt{X}$ or $X = \frac{1}{\sqrt{X}}$?

Reply:
the latter.
I just found that I named it wrong :(